test/hotspot/jtreg/compiler/vectorization/runner/BasicFloatOpTest.java

```diff
@@ -50,11 +51,72 @@ public class BasicFloatOpTest extends VectorizationTestRunner {
     private float[] c;
+    private float[] d;
+    private float[] e;
 
     public BasicFloatOpTest() {
+        // Positive test values                       sign |   exponent | mantisa
+        float smallPositive   = Float.intBitsToFloat(0<<31 | 0x3f << 23 | 0x30000f);
+        float positive        = Float.intBitsToFloat(0<<31 | 0x7f << 23 | 0x30000f);
+        float bigPositive     = Float.intBitsToFloat(0<<31 | 0x7f << 23 | 0x30100f);
+        float biggerPositive  = Float.intBitsToFloat(0<<31 | 0xfe << 23 | 0x30000f);
+        float maxPositive     = Float.MAX_VALUE;
+
+        // Special positive
+        float nan1  = Float.intBitsToFloat(0<<31 | 0xff << 23 | 0x7fffff);
+        float nan2  = Float.intBitsToFloat(0<<31 | 0xff << 23 | 0x30000f);
+        float inf   = Float.intBitsToFloat(0<<31 | 0xff << 23);
+        float zero  = 0.0f;
+
+        // Negative test values                       sign |   exponent | mantisa
+        float smallNegative   = Float.intBitsToFloat(1<<31 | 0x03 << 23 | 0x30000f);
+        float negative        = Float.intBitsToFloat(1<<31 | 0x83 << 23 | 0x30100f);
+        float bigNegative     = Float.intBitsToFloat(1<<31 | 0x83 << 23 | 0x30000f);
+        float biggerNegative  = Float.intBitsToFloat(1<<31 | 0x86 << 23 | 0x30000f);
+        float maxNegative     = Float.intBitsToFloat(1<<31 | 0xfe << 23 | 0x7fffff);
+
+        // Special negative
+        float nNan1  = Float.intBitsToFloat(1<<31 | 0xff << 23 | 0x7fffff);
+        float nNan2  = Float.intBitsToFloat(1<<31 | 0xff << 23 | 0x30000f);
+        float nInf   = Float.intBitsToFloat(1<<31 | 0xff << 23);
+        float nZero  = -0.0f;
+
+        float[] orderedList = new float[] {
+            nInf, maxNegative, biggerNegative, bigNegative, negative, smallNegative, nZero,
+            zero, smallPositive, positive, bigPositive, biggerPositive, maxPositive, inf
+        };
+
+        float[] NaNs = new float[] {
+            nan1, nan2, nNan1, nNan2
+        };
+
+        float[] numberList = new float[] {
+            nInf, maxNegative, biggerNegative, bigNegative, negative, smallNegative, nZero,
+            zero, smallPositive, positive, bigPositive, biggerPositive, maxPositive, inf,
+            nan1, nan2, nNan1, nNan2
+        };
+
+        Random rnd = new Random(11);
         a = new float[SIZE];
         b = new float[SIZE];
         c = new float[SIZE];
+        d = new float[SIZE];
+        e = new float[SIZE];
+
+        for (int i = 0; i < SIZE;) {
+            for (int j = 0; j < numberList.length && i < SIZE; j++, i++) {
+                for (int k = j; k < numberList.length && i < SIZE; k++, i++) {
+                    if (rnd.nextBoolean()) {
+                        d[i] = numberList[j];
+                        e[i] = numberList[k];
+                    } else {
+                        d[i] = numberList[k];
+                        e[i] = numberList[j];
+                    }
+                }
+            }
+        }
+
```

Comment:
Test performs IR validation, you can also update existing functional test with more test values.
test/hotspot/jtreg/compiler/intrinsics/math/TestFpMinMaxIntrinsics.java

Reply:
Same as above, I am ok to remove this change to the test too.. I didn't know where the other tests where and by the time I did find those, already added this. (Figured "more test === good", but its just duplicate)